scripts/validate-vale-rules.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
Hey, validate-vale-rules.sh counts the lines of testinvalid.adoc and compares it to the amount of vale alerts for testinvalid.adoc. If there are more lines than Vale alerts, it fails as it expects 1 alert per line. My rules are multi-line, so they script ain't gonna work. If you want to remove this folder I can merge the other changes

Reply:
hmmm. that's a bummer. wonder how we can change this to work?

Reply:
We can mark tests with comments in the fixtures and just count the comments. I'll do that Monday.